### PR TITLE
Feature/as 1 lpd task list

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-1

## Description of change
Initial setup of LPA Questionnaire application and task list page scaffolding

## Checklist

- [X] Requires infrastructure changes
- [X] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
I need to check that this works in the K8S setup before merging

File: packages/lpa-questionnaire-web-app/src/views/includes/head.njk
Comment (a6software) [inline packages/lpa-questionnaire-web-app/src/views/includes/head.njk:1]: there's a lot of debate around this, so heads up, this may not remain as is

File: packages/lpa-questionnaire-web-app/src/views/layouts/main.njk
Comment (a6software) [inline packages/lpa-questionnaire-web-app/src/views/layouts/main.njk:36]: there's debate around this as well, not sure what but this might not be right.

**Review by a6software — APPROVED** [review]
Seems fine to me, but there's a lot of duplication.

I can see pros and cons to this, but if happy with that, all good from me.

File: packages/lpa-questionnaire-web-app/src/views/includes/head.njk
Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/src/views/includes/head.njk:1]: can you keep us updated or if we can be party to discussions that would be helpful. If you can give more details as to what is up for debate that would help also.

File: packages/lpa-questionnaire-web-app/src/views/layouts/main.njk
Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/src/views/layouts/main.njk:36]: As above, need some more info and need to be kept up to date on this

File: packages/lpa-questionnaire-web-app/src/views/includes/head.njk
Comment (a6software) [reply packages/lpa-questionnaire-web-app/src/views/includes/head.njk:1]: Sure - I'm not actively working on GA stuff so this was to flag that there's a potential for this to change but I may not have immediate visibility on it.

The gist is that this code drops a GA cookie on every user, whereas we should probably have a cookie banner which needs accepting before we drop the cookie. That's all still up for debate afaik, but if it impacts us, it will likely impact you, too. 

No action at this time ofc, just a heads up.

File: packages/lpa-questionnaire-web-app/src/views/layouts/main.njk
Comment (a6software) [reply packages/lpa-questionnaire-web-app/src/views/layouts/main.njk]: Jamie Crisp (from PINS) is the person to ask about this, he is the UX lead for these projects. 

I don't know what it is they want here, but they have mentioned we copied this from their prototype and it's not the expected behaviour. I've not seen a ticket for what is the expected behaviour, but I believe this will need to change.

Pretty much same issue as GA. I'm not working on it, so I might not have immediate visibility when the requirement changes, it's more of a heads up.

I'll raise both of these issues internally so the entire team is aware.

File: packages/lpa-questionnaire-web-app/src/views/includes/head.njk
Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/src/views/includes/head.njk:1]: That's great, thanks for the extra detail. Will leave this as-is for now

File: packages/lpa-questionnaire-web-app/src/views/layouts/main.njk
Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/src/views/layouts/main.njk]: OK thanks, I believe we're talking to Jamie around the tickets ourselves so will bring that up with him.

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
image
Can the commit history be tidied up a bit before moving into `master`? It feels like the ones with arrows should be merged today (happy for a conversation on this). In any case, the spelling of "improve" should be amended.

From a code PoV, I'm happy with the infra/deployment changes I made yesterday as they worked fine. I'll leave final say on tests and web app to @danWatkinson and @a6software

**Comment by MrSimonEmms** [comment]
Also, @danWatkinson can you investigate with @davehlewis if the Cypress test failures are an actual problem, or just the usual timeout we're experiencing?

**Comment by davehlewis** [comment]
@MrSimonEmms have made the suggested commit changes, happy to pass this from your side of things?

**Review by danWatkinson — APPROVED** [review]
@MrSimonEmms it's unfortunate that this e2e-test instability has cropped up as we're trying to onboard a new team.
right now: the set of tests that keep getting stuck represent our 'queued up release 2' work, not the things we are currently releasing to production, so I'm happy that there is nothing in there that is failing and causing an issue. the ACP tests failing would concern me more.

**Review by MrSimonEmms — APPROVED** [review]

**Comment by MrSimonEmms** [comment]
When this has been merged, please also merge #406

**Review by a6software — APPROVED** [review]
Approving with the caveat that the Cypress pipeline is currently failing, but to the very best of my knowledge, that is unrelated to this PR and shouldn't be a blocker. 

Please feel free to overrule that opinion but from me it's a 👍

**Comment by MrSimonEmms** [comment]
> Approving with the caveat that the Cypress pipeline is currently failing, but to the very best of my knowledge, that is unrelated to this PR and shouldn't be a blocker.
> 
> Please feel free to overrule that opinion but from me it's a +1

I concur

**Pull request merged**
